src/mmw/js/src/analyze/views.js

```diff
@@ -30,6 +33,50 @@ var ResultsView = Marionette.LayoutView.extend({
         analyzeRegion: '#analyze-tab-contents'
     },
 
+    ui: {
+        'modelPackageLinks': 'a.model-package',
+    },
+
+    events: {
+        'click @ui.modelPackageLinks': 'selectModelPackage',
+    },
+
+    selectModelPackage: function (e) {
+        e.preventDefault();
+
+        var modelPackages = settings.get('model_packages'),
+            modelPackageName = $(e.target).data('id'),
+            modelPackage = _.find(modelPackages, {name: modelPackageName}),
+            newProjectUrl = '/project/new/' + modelPackageName,
+            projectUrl = '/project';
+
+        if (!modelPackage.disabled) {
+            if (settings.get('itsi_embed') && App.currentProject && !App.currentProject.get('needs_reset')) {
+                var currModelPackageName = App.currentProject && App.currentProject.get('model_package');
+                if (modelPackageName === currModelPackageName) {
+                    // Go to existing project
+                    router.navigate(projectUrl, {trigger: true});
+                } else {
+                    var confirmNewProject = new modalViews.ConfirmView({
+                        model: new modalModels.ConfirmModel({
+                            question: 'If you change the model package you will lose your current work.',
+                            confirmLabel: 'Switch Model Package',
```

Comment:
We might have called it modelPackage to avoid confusion with the concept of a Backbone model. I can change it in the dialog though.